src/nvim/event/process.c

```diff
@@ -244,12 +242,16 @@ void process_stop(Process *proc) FUNC_ATTR_NONNULL_ALL
       abort();
   }
 
+  ILOG("Sending SIGTERM to pid %d", proc->pid);
+  uv_kill(proc->pid, SIGTERM);
+
   Loop *loop = proc->loop;
   if (!loop->children_stop_requests++) {
     // When there's at least one stop request pending, start a timer that
-    // will periodically check if a signal should be send to a to the job
+    // will periodically check if a signal should be send to the job.
     DLOG("Starting job kill timer");
-    uv_timer_start(&loop->children_kill_timer, children_kill_cb, 100, 100);
+    uv_timer_start(&loop->children_kill_timer, children_kill_cb,
+            KILL_TIMEOUT_MS, 100);
```

Comment:
@jamessan https://travis-ci.org/neovim/neovim/builds/228561905 has green status in the first build.